deployments/container/vulkan/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM nvcr.io/nvidia/cuda:12.8.1-base-ubuntu22.04 AS builder
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
+    wget \
+    && \
+    rm -rf /var/lib/apt/lists/*
+
+# See instructions from https://vulkan.lunarg.com/doc/sdk/1.4.309.0/linux/getting_started_ubuntu.html
+RUN wget -qO- https://packages.lunarg.com/lunarg-signing-key-pub.asc | tee /etc/apt/trusted.gpg.d/lunarg.asc \
+    && \
+    wget -qO /etc/apt/sources.list.d/lunarg-vulkan-jammy.list http://packages.lunarg.com/vulkan/lunarg-vulkan-jammy.list \
+    && \
+    apt update -y \
+    && \
+    apt install -y --no-install-recommends vulkan-sdk \
+    && \
+    rm -rf /var/lib/apt/lists/*
+
+# TODO: Should we include this in the list above.
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git \
+    make \
+    cmake \
+    pkg-config \
+    gcc \
+    g++ \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /build
+
+ARG BUILD_EXAMPLES="computeheadless renderheadless"
+ENV BUILD_EXAMPLES=$BUILD_EXAMPLES
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libglm-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# TODO: We should update to the official samples
```

Comment:
Curious, why are we not using the official ones in this build?

Reply:
I did not want to change how the samples are being built when compared to https://gitlab.com/nvidia/container-images/vulkan/-/blame/master/samples/Dockerfile.ubuntu?ref_type=heads#L35 in this commit.